src/puppetlabs/trapperkeeper/services/status/status_logging.clj

```diff
@@ -0,0 +1,22 @@
+(ns puppetlabs.trapperkeeper.services.status.status-logging
+  (:require [clojure.tools.logging :as log]
+            [schema.core :as schema]
+            [schema.utils :refer [validation-error-explain]]
+            [cheshire.core :as json]
+            [puppetlabs.trapperkeeper.services.status.status-core :as status-core])
+  (:import (clojure.lang IFn)
+           (java.util.concurrent Future)))
+
+(schema/defn start-background-task :- Future
+  "Creates a future which calls callback every frequency milliseconds"
+  [frequency :- schema/Int
+   callback :- IFn]
+  (future (while true
+            (callback)
+            (Thread/sleep frequency))))
+
+(defn logging-fn
+  "Log status information at the debug level as json"
+  []
+  (let [status (status-core/v1-status :debug)]
```

Comment:
I notice you're logging at debug here.  That seems like the best level if it works... but... have you tested this with your `<root>` logger configured to log at DEBUG level?  And validated that these messages don't show up in the root logger?  Just wanting to make sure that they weren't being filtered by the level.